src/course_bot/general.clj

```diff
@@ -81,8 +83,10 @@
 (defn send-whoami
   ([tx token id] (send-whoami tx token id id))
   ([tx token id about]
-   (let [{name :name group :group} (codax/get-at tx [about])]
-     (talk/send-text token id (format (tr :general/who-am-i-3) name group about)))))
+   (let [{name :name group :group pres :presentation} (codax/get-at tx [about])
+         lab1                                         (-> pres :lab1)
+         grouplab1                                    (-> lab1 :group)]
```

Comment:
In theory, csa.clj can specify several presentations for the course. So we need to pass it as an argument to `whoami-talk` command. Like:

```clj
      (general/whoami-talk db conf (pres/group "lab1") (pres/group "labN"))
```

The output of `pres/group` should be connected to the end of normal whoami messages